### ISSUE TITLE
With the latest build, log is not monitored

### ISSUE THREAD

**Issue by belkiss** [hillin/FASTBuilder#4]
Hello!
With the latest build, the window remains empty if it's opened after the build has started.
It displays everything if it's started after the build has finished though, but starting a build won't make it appear, even restarting the app.
It feels like the file is not monitored...

I seem to be missing some nuget package so I cannot compile locally yet though, I need to investigate why.

**Comment by hillin** [comment]
Hi,

I'm observing this issue for several days but cannot reproduce it. Could you please try to reproduce and debug it?

**Comment by belkiss** [comment]
Yes, I managed to get a repro, it's due to an uncaught exception in GetRootProcessId, when calling
`if (process.StartTime > startTime)`

I get:
```
Exception thrown: 'System.ComponentModel.Win32Exception' in System.dll
Access is denied
```

Because the process points to svchost

**Comment by belkiss** [comment]
A big and ugly `try/catch{ return -1; }` around the accesses to `process.` make the problem go away.
So line:
https://github.com/hillin/FASTBuild-Dashboard/blob/aac7078f5682c0bbee0d85ba7e0dad5ed1c71675/Source/ViewModels/Build/BuildInitiatorProcessViewModel.cs#L142
and:
https://github.com/hillin/FASTBuild-Dashboard/blob/aac7078f5682c0bbee0d85ba7e0dad5ed1c71675/Source/ViewModels/Build/BuildInitiatorProcessViewModel.cs#L147

**Comment by hillin** [comment]
Yeah I should definitely be more cautious when dealing with these privilege related things. I'll fix it. Thanks!

**Comment by hillin** [comment]
I hope this can be fixed with 7f3aa75cd016bbd2a6f931abbf582353a13c4476.

**Comment by belkiss** [comment]
Yep! Looks good now! Thanks!
